opennms-web-api/src/main/java/org/opennms/web/svclayer/support/DefaultGraphResultsService.java

```diff
@@ -349,5 +382,12 @@ public void setRrdDao(RrdDao rrdDao) {
     public void setEventProxy(EventProxy eventProxy) {
         m_eventProxy = eventProxy;
     }
-    
+
+    /**
```

Comment:
Delete the javadoc here as it is implied

Reply:
Just made it look similar to the other methods in the Class.